lib/linguist/languages.yml

```diff
@@ -824,6 +824,18 @@ C2hs Haskell:
   codemirror_mode: haskell
   codemirror_mime_type: text/x-haskell
   language_id: 45
+C3:
+  type: programming
+  color: "#7C3AED"
+  extensions:
+  - ".c3"
+  - ".c3i"
+  - ".c3t"
```

Comment:
We need search result links in the template for all extensions individually. Three extensions means three links, and they each need to meet our usage requirements individually.

I recommend removing any that are no where near meeting our requirements.

Reply:
> We need search result links in the template for all extensions individually. Three extensions means three links, and they each need to meet our usage requirements individually.
> 
> I recommend removing any that are no where near meeting our requirements.

I'll add it though.

Reply:
Your `.c3t` search returns zero results. This will hold back this PR. We look at extensions in isolation so each needs to meet our requirements before the PR is merged. Once Linguist knows about a language, users can use an override for less popular extensions until they meet usage requirements and are added.